client/src/components/RequestInfo/MyModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Grid, Typography, Avatar } from '@mui/material';
+import Modal from '@mui/material/Modal';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+import useStyles from './useStyles';
+
+const MyModal: React.FC<{
+  dummyData: any;
+  open: boolean;
+  handleClose: () => void;
+}> = ({ dummyData, open, handleClose }) => {
+  const classes = useStyles();
+
+  return (
+    <Modal open={open} onClose={handleClose} disableAutoFocus={true}>
+      <Box className={classes.modalBox}>
+        <Typography id="modal-modal-title" variant="h6" component="h2"></Typography>
```

Comment:
This `Typography` component has no text within it, let's remove it or add the appropriate title here